funfact/model/_factorization.py

```diff
@@ -133,6 +134,14 @@ def ndim(self):
         '''The dimensionality of the result tensor.'''
         return self.tsrex.ndim
 
+    @property
+    def penalty(self):
+        '''The penalty of the result tensor.'''
+        _tsrex = self.tsrex | PenaltyEvaluator()
```

Comment:
No need for the `_` prefix here since these variables are local within the scope of the method. :smile: